homeassistant/components/tessie/sensor.py

```diff
@@ -0,0 +1,203 @@
+"""Sensor platform for Tessie integration."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    PERCENTAGE,
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfLength,
+    UnitOfPower,
+    UnitOfPressure,
+    UnitOfSpeed,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN, TessieStatus
+from .coordinator import TessieDataUpdateCoordinator
+from .entity import TessieEntity
+
+PARALLEL_UPDATES = 0
+
```

Comment:
Please add entity_category to entities to non-primary entities.
https://developers.home-assistant.io/docs/core/entity#generic-properties